eng/pipelines/runtimelab.yml

```diff
@@ -101,3 +101,7 @@ stages:
             isOfficialBuild: true
             librariesBinArtifactName: libraries_bin_official_allconfigurations
 
+- ${{ if eq(variables.isOfficialBuild, true) }}:
+  - template: /eng/pipelines/official/stages/publish.yml
+      parameters:
+            isOfficialBuild: ${{ variables.isOfficialBuild }}
```

Comment:
this indentation is off by one tab.

Reply:
I'll take a look. I tried to mimic the original file but it probably got messed up in the copy.